### PR TITLE
✨ [AMP-Digiteka] ADD a new vendor Digiteka

### PR DESCRIPTION
**Pull request by ambrugeat**
ADD a new vendor Digiteka

### CODE REVIEW

File: ads/outstream.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+document.write('<script type="text/javascript">ULTIMEDIAOUTSTREAM_target = "ultimedia-outstream-player";ULTIMEDIAOUTSTREAM_mdtk = "01540779";ULTIMEDIAOUTSTREAM_zone = 1;</script>');
```
Comment (calebcordry) [inline ads/outstream.js:1]: Can you explain whats going on with this file?

File: ads/vendors/digiteka.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/digiteka.js:2]: nit: 2021

File: ads/vendors/digiteka.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```
Comment (calebcordry) [inline ads/vendors/digiteka.md:2]: nit: 2021

File: ads/_config.js [hunk 4]
```diff
@@ -1195,6 +1195,8 @@ const adConfig = jsonConfiguration({
 
   'triplelift': {},
 
+  'digiteka': {},
```
Comment (calebcordry) [inline ads/_config.js:1198]: I see you have not implemented renderStart, it is up to you but should give you some performance gains.

**Review by calebcordry — CHANGES_REQUESTED** [review]
Thanks for contributing! Let me know if you need help figuring out the CLA process.

File: ads/_config.js (hunk 4 above)
Comment (marcDigiteka) [reply ads/_config.js:1198]: I'll check it out then, thx

**Comment by marcDigiteka** [comment]
I have made the proper changes.
Is it OK now ?

File: ads/vendors/digiteka.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+</amp-ad>
+```
+
+## Configuration
+
+For configuration semantics, please [contact Digiteka](http://digiteka.com).
+
+Supported parameters:
+
+-   `src`
```
Comment (calebcordry) [inline ads/vendors/digiteka.md:40]: I don't see src in your example? Also could you add `data-zone` & `data-mdtk`? Thanks

Comment (marcDigiteka) [reply ads/vendors/digiteka.md]: Modification has been made

**Review by calebcordry — APPROVED** [review]

**Comment by marcDigiteka** [comment]
Why am I getting a Failing test on ampproject/tests/e2e (nomodule) when the gulp pr-check is perfectly fine !

I checked out what failed and it's a amp-base-carousel error => I haven't changed any file for this... :/

https://app.circleci.com/pipelines/github/ampproject/amphtml/4500/workflows/3112f118-8322-4bd5-a6f5-abd003217cf6/jobs/58700

**Comment by calebcordry** [comment]
Yeah sorry that was just a flaky test, I think it has been skipped on master. Could you try rebasing? I think it will fix the tests. Let me know if you need assistance.

**Comment by marcDigiteka** [comment]
Hello ! :)
Just for my information, how long does it take for a PR to be merged ?
Thanks a lot

**Comment by calebcordry** [comment]
Just saw that the tests had passed. Thanks for contributing!

**Comment by marcDigiteka** [comment]
Hi @calebcordry !
I'm very new to this.
How long does it take for the new type to be available in this file : https://cdn.ampproject.org/v0/amp-ad-0.1.js

Is it not supposed to be live as it is merged ?
Thanks a lot

**Pull request merged**